api/pkg/service/resource/resource.go

```diff
@@ -346,6 +371,28 @@ func filterByKind(t string) func(db *gorm.DB) *gorm.DB {
 	}
 }
 
+func filterByKinds(t []string) func(db *gorm.DB) *gorm.DB {
+	if len(t) == 0 {
+		return noop
+	}
+
+	return func(db *gorm.DB) *gorm.DB {
+		t = lower(t)
+		return db.Where("LOWER(kind) IN (?)", t)
+	}
+}
+
+func filterByTags(t []uint) func(db *gorm.DB) *gorm.DB {
```

Comment:
this is actually filter by id, we are passing resource ids... should we rename it to `filterByIds`